tasks/main.yml

```diff
@@ -115,6 +115,14 @@
   with_dict: tinc_external_hosts
   when: tinc_external_hosts is defined and tinc_external_hosts
 
+- name: check systemd
+  shell: "cat /proc/1/comm"
+  ignore_errors: yes
```

Comment:
I don't accept `ignore_errors: Tru` in roles - you should handle the errors, not ignore them. For this check, you could test if `/run/systemd/system` directory is present, it's a pretty good indicator that a given host is running using `systemd`.

Also, if you only set up the unit files, you can do it without checking if `systemd` is present; default Wheezy installation contains `/etc/systemd/` directory with proper structure, you can just put units there regardless. They will be used when needed.